awx/ui/src/screens/Job/JobOutput/JobOutput.js

```diff
@@ -196,14 +198,28 @@ function JobOutput({ job, eventRelatedSearchableKeys, eventSearchableKeys }) {
     rebuildEventsTree();
   }, [isFlatMode]); // eslint-disable-line react-hooks/exhaustive-deps
 
+  const pollForEventsProcessed = useCallback(async () => {
```

Comment:
Does this function poll at increasing longer intervals?  Like it starts polling at 250, and then doubles its interval each time until it gets to 1000 * 60 * 5 (5 minutes)?

Reply:
yes, exactly